src/components/userProfile/components/SideMenu.js

```diff
@@ -15,11 +16,18 @@ export default class SideMenu extends React.Component {
                 className={MENU}
                 theme={DARK}
                 mode={INLINE}
+                selectedKeys={[location.pathname]}
             >
                 {
-                    SIDE_MENU_ITEMS.map(({ iconType, label }) => (
-                        <Menu.Item key={label}>
-                            <span><Icon type={iconType} />{label}</span>
+                    SIDE_MENU_ITEMS.map(({ iconType, label, link }) => (
+                        <Menu.Item key={link || label}>
+                            {
+                                link ?
+                                    <AppLink to={link}>
+                                        <span><Icon type={iconType} />{label}</span>
+                                    </AppLink> :
+                                    <span><Icon type={iconType} />{label}</span>
+                            }
```

Comment:
When all the side menu items have a link would it still be necessary to use ternary?

Reply:
nope it would not be